components/button/index.html

```diff
@@ -11,10 +11,13 @@
 </head>
     <body>
         <button class="button button_primary">
-            Primary
+            <span class="button_text">Primary</span>
         </button>
-        <button class="button button_primary" disabled>
-            Primary disabled
+        <!-- <button class="button button_primary" disabled>
```

Comment:
Designs are updated and the 'disabled' state is available for development now. Uncomment 'disabled' state and finish it, please.